docs/_news/news11.md

```diff
@@ -56,6 +58,9 @@ We've continued to work hard on getting the IntelliJ experience up and consisten
 - You can [view the pod name and namspace for remote projects in the **Project Overview** page](https://github.com/eclipse/codewind/issues/1525).
 - You can [select multiple entries in the table of template sources in the **Manage Template Sources** wizard](https://github.com/eclipse/codewind/issues/2353).
 
+##### IntelliJ
+[The status representation indicator next to the project in the Codewind Explorer view is improved](https://github.com/eclipse/codewind/issues/159).
```

Comment:
Maybe remove "next to the project". I don't think the phrase is necessary since we also specify the Codewind Explorer view, and I'm wondering if it might be confusing for a user who uses a screen reader.